common/types/double.go

```diff
@@ -127,17 +126,17 @@ func (d Double) ConvertToNative(typeDesc reflect.Type) (interface{}, error) {
 func (d Double) ConvertToType(typeVal ref.Type) ref.Val {
 	switch typeVal {
 	case IntType:
-		i := math.Round(float64(d))
```

Comment:
The removal of the rounding behavior on conversion is expected and part of a cel-spec update.